src/scripts/oauth_tables_script.sql

```diff
@@ -0,0 +1,110 @@
+
+DROP TABLE IF EXISTS app_clients CASCADE;
+
+CREATE TABLE app_clients(
+             id BIGSERIAL NOT NULL,
+             client_name VARCHAR(50) UNIQUE NOT NULL,
+             client_id VARCHAR(50) UNIQUE NOT NULL,
+             client_secret VARCHAR(50) UNIQUE NOT NULL,
+             scope_id BIGINT NOT NULL,
+             expiration_time TIMESTAMP WITHOUT TIME ZONE,
```

Comment:
You need to handle both a resfresh token and an access token expires_in fields. This field is measured in seconds

Reply:
done. i changed datatype to bigint to manage seconds.